tests/migrations/test_autodetector.py

```diff
@@ -1515,7 +1515,11 @@ def test_alter_field_to_not_null_with_db_default(self, mocked_ask_method):
             changes, "testapp", 0, 0, name="name", preserve_default=True
         )
         self.assertOperationFieldAttributes(
-            changes, "testapp", 0, 0, db_default=models.Value("Ada Lovelace")
+            changes,
+            "testapp",
+            0,
+            0,
+            db_default="Ada Lovelace",
```

Comment:
This one surprised me as the exact same call above is on a single line but black doing weird things I guess 🤷

Reply:
You added a trailing comma. If you remove it black will format it on one line.

Reply:
Thanks, that was it! I must have added the comma when iterating through the patch.